docs/source_config_options.rst

```diff
@@ -1,6 +1,19 @@
 JDBC Source Configuration Options
 ---------------------------------
 
+Security
+^^^^^^^^
+
+In the connector configuration you will notice there are no security parameters. This is because
```

Comment:
if you're going to repeat text, it is possible to include the same text via an [include](http://docutils.sourceforge.net/docs/ref/rst/directives.html#include). @joel-hamill recently started using these to make repeated text more manageable. seems reasonable to do here, and he might have more guidance on how it makes best sense to manage this going forward.

Reply:
yeah this makes sense, I gave the ``include`` directive a shot in the latest commit. I can already see just in making a couple of changes how mistake prone this repeated text would be if copied in multiple places. If it looks ok to @joel-hamill then I'll merge.